### DISCUSSION TITLE
Singleplayer storyline, but in multiplayer?

### DISCUSSION THREAD

**Discussion by seanbrockest** [diasurgical/DevilutionX#7566] [Q&A] [answered]
The one thing that irked me back in... oh god was it really 1996? damnit I'm old..... I might need a nap.

Anyway. The one thing that I hated about my "favorite game ever!" was that you couldn't do the single player campaign with a friend. There were some awesome quests, but they were almost entirely gone from the multiplayer experience.

Is there any way to bring that into the multiplayer game with modding?

**Comment by julealgon** [comment, marked as answer]
@seanbrockest check the issues and other discussions in the repository. This has been happening for a while now. Quests now work in multiplayer already.

Check the release notes for 1.5
- https://github.com/diasurgical/devilutionX/releases/tag/1.5.0

> Multiplayer
> - All quests can now be played, enabled from settings menu

**Reply by seanbrockest** [reply]
Awesome! I did a search, but it came up with nothing. I'm really not a github guy.

Thanks!

**Comment by AJenbo** [comment]
> oh god was it really 1996?

Probably not, the game was available on the last day of 1996 so unlikely you got much game time in there :)

**Reply by seanbrockest** [reply]
Oh thank the dark lords, that makes me feel so much younger....
